packages/select/test/queryListTests.tsx

```diff
@@ -89,6 +95,8 @@ describe("<QueryList>", () => {
             filmQueryList.setState({ query: "query" });
             filmQueryList.setState({ activeItem: undefined });
             assert.equal(testProps.onActiveItemChange.callCount, 0);
+
+            filmQueryList.unmount();
```

Comment:
Mounted components were never being unmounted in these unit tests

Reply:
was this a problem? i've found you only need to unmount when you use the `attachTo` option to actually render into the document (which is not used here).

Reply:
I've been in the habit of always unmounting. I seem to remember enzyme documentation recommending always unmounting, but current documentation says: "Keep that in mind while writing your tests and, if necessary, use .unmount() or something similar as cleanup."

I can remove the unmount calls here since they technically aren't necessary, but I'll continue to always unmount as a rule in my projects so that I don't get obscure problems in the situations where unmounting actually is important.

Reply:
sounds good. thanks.